web/src/main/java/com/microsoft/applicationinsights/web/internal/WebRequestTrackingFilter.java

```diff
@@ -59,20 +58,24 @@
     private boolean agentIsUp = false;
     private final LinkedList<ThreadLocalCleaner> cleaners = new LinkedList<ThreadLocalCleaner>();
 
+    //Only for Spring Boot Application (External Name Setting)
```

Comment:
I don't think we need to limit use of it for Spring Boot. May one decide to do it in regular app?